adapters/resetdigital/resetdigital_test.go

```diff
@@ -1,21 +1,1004 @@
 package resetdigital
 
 import (
+	"encoding/json"
+	"net/http"
 	"testing"
 
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v3/adapters"
 	"github.com/prebid/prebid-server/v3/adapters/adapterstest"
 	"github.com/prebid/prebid-server/v3/config"
+	"github.com/prebid/prebid-server/v3/errortypes"
 	"github.com/prebid/prebid-server/v3/openrtb_ext"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJsonSamples(t *testing.T) {
```

Comment:
We require using the JSON test framework to achieve code coverage wherever possible as described in the docs: [Adapter Code Tests](https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#adapter-code-tests). The framework performs additional checks that you would not otherwise get with a traditional unit test like ensuring your adapter isn't corrupting shared memory. Unit tests should be used sparingly or not at all as all code coverage is usually attainable via JSON tests.